assets/scss/_explorer.scss

```diff
@@ -1,9 +1,48 @@
 .explorer {
 
-  &.pixelsets {
+  aside.sidebar-left {
+    @include xy-cell($size: 2.5, $gutter-output: false);
+
+    .filters {
+
+      form {
+        .field {
+          label {
+            font-weight: bold;
+
+            ul {
+              max-height: 200px;
+              margin-left: 0;
+              list-style: none;
+              overflow-y: auto;
+
+              &#id_tags {
```

Comment:
mmh, it is an id here? why do you need to nest it? it should be unique and used once.

Reply:
Oops. My bad. That's silly.

Reply:
no problemo